src/features/import/components/Importer/validation/index.tsx

```diff
@@ -0,0 +1,181 @@
+import { Typography } from '@mui/material';
+import { useState } from 'react';
+import { Box, Stack } from '@mui/system';
+
+import AddedTagsTracker from './AddedTagsTracker';
+import { getOrgsStates } from 'features/import/utils/getOrgsStates';
+import ImportAlert from './importAlert';
+import ImportChangeTracker from './importChangeTracker';
+import { Msg } from 'core/i18n';
+import useAlertsStates from 'features/import/hooks/useAlertsStates';
+import { useNumericRouteParams } from 'core/hooks';
+import PeopleCounter, { COUNT_STATUS } from './PeopleCounter';
+
+import messageIds from 'features/import/l10n/messageIds';
+
+export interface FakeDataType {
+  summary: {
+    membershipsCreated: {
+      byOrganization: {
+        [key: number]: number;
+      };
+      total: number;
+    };
+    peopleCreated: {
+      total: number;
+    };
+    peopleUpdated: {
+      byField: {
+        [key: string]: number;
+      };
+      total: number;
+    };
+    tagsCreated: {
+      byTag: {
+        [key: number]: number;
+      };
+      total: number;
+    };
+  };
+}
+
+interface ValidationProps {
+  onClickBack: () => void;
+  onDisabled: (value: boolean) => void;
+}
+
+const Validation = ({ onClickBack, onDisabled }: ValidationProps) => {
+  const fake = {
+    summary: {
+      membershipsCreated: {
+        byOrganization: {
+          1: 10,
+          2: 10,
+          4: 10,
+          7: 10,
+        },
+        total: 60,
+      },
+      peopleCreated: {
+        total: 60,
+      },
+      peopleUpdated: {
+        byField: {
+          alt_phone: 20,
+          city: 2,
+          date_of_birth: 25,
+          email: 10,
+          first_name: 25,
+          gender: 3,
+          join_date: 20,
+          last_name: 20,
+          zip_code: 2,
+        },
+        total: 100,
+      },
+      tagsCreated: {
+        byTag: {
+          11: 20,
+          12: 20,
+          9: 20,
+        },
+        total: 60,
+      },
+    },
+  };
+  const [checkedIndexes, setCheckedIndexes] = useState<number[]>([]);
+  const { orgId } = useNumericRouteParams();
+  const alertStates = useAlertsStates(fake.summary, orgId);
+
+  const orgsStates = getOrgsStates(fake.summary.membershipsCreated);
+
+  const warningAlerts = alertStates.filter(
+    (item) => item.alertStatus === 'warning'
+  );
+  const errorExists =
+    alertStates.filter((item) => item.alertStatus === 'error').length > 0;
+
+  if (
+    (warningAlerts.length === checkedIndexes.length && !errorExists) ||
+    alertStates.filter((item) => item.alertStatus === 'info').length > 0
```

Comment:
i think you should use the enum `ALERT_STATUS` you created to compare with instead of string values here. Or is there a special reason that you use string values here and not the enum?